bugtool/options/options.go

```diff
@@ -37,7 +37,8 @@ type Config struct {
 	ExcludeObjectFiles bool          `mapstructure:"exclude-object-files"`
 	Generate           bool          `mapstructure:"generate"`
 	Timeout            time.Duration `mapstructure:"timeout"`
-	Debug              bool
+	Debug              bool          `mapstructure:"debug"`
+	Wait               bool          `mapstructure:"wait"`
```

Comment:
Does this need a separate timeout? Or is the general timeout sufficient?

Reply:
Good idea 👍